src/helpers/request-helper.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios'
+
+/**
+ * Performs a HTTP GET request and returns a promise that will be resolved by function's clients
+ * @param {string} url - The URL to be used on request
+ * @param { Object } headers - The header used on request
+ * @returns { Promise }
+ */
+ export async function get(url, headers = {}) {
+  return await axios.get(url, {
```

Comment:
I think that you don't need this `await`, because you don't need keep waiting the promise resolve.
Just delegate this responsibility to who to call the function.

```suggestion
  return axios.get(url, {
```

Reply:
I had delegated this to all functions (laughs)
But there is no need to do so.
Go it.